tools/CMakeLists.txt

```diff
@@ -1,44 +1,64 @@
 # Copyright (C) 2018-2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
+
 cmake_minimum_required(VERSION 3.13)
 
-project(python_tools)
+project(OpenVINO_Tools DESCRIPTION "OpenVINO toolkit Development Tools")
 
 if(NOT DEFINED OpenVINO_SOURCE_DIR)
     find_package(InferenceEngineDeveloperPackage QUIET)
+    set(python_tools_only ON)
+endif()
+
+# C++ tools
+
+if(NOT python_tools_only)
+    add_subdirectory(compile_tool)
 endif()
 
+# Python tools
+
+# install deployment_manager
+
+ie_cpack_add_component(deployment_manager REQUIRED)
+install(DIRECTORY deployment_manager/
+        DESTINATION deployment_tools/tools/deployment_manager
+        COMPONENT deployment_manager
+        USE_SOURCE_PERMISSIONS)
+
 if(ENABLE_PYTHON)
     find_package(PythonInterp 3 REQUIRED)
     set(PYTHON_VERSION python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
 
-    set(TARGET_NAME "python_tools")
-
     if(WIN32)
         set(PYTHON_BRIDGE_OUTPUT_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/$<CONFIG>/python_api/${PYTHON_VERSION}/openvino)
     else()
         set(PYTHON_BRIDGE_OUTPUT_DIRECTORY ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/python_api/${PYTHON_VERSION}/openvino)
     endif()
 
     # creates a copy inside bin directory for developers to have ability running python benchmark_app
-    add_custom_target(${TARGET_NAME} ALL
+    add_custom_target(python_tools ALL
```

Comment:
This action copy benchmark tool to a build directory (as well as ie python api & ngraph api), so it could be used after the build by adding directory to PYHTONPATH. Do we plan to continue providing this way along with`setup.py install`?

At least it would be nice to rename action to something specific, like `python_tools_benchmark`.

Reply:
renamed to `python_tools_benchmark`